### PR TITLE
Validation added in addToCart & Removed duplicate ```stock``` key for variants from ProductSeeder

### PR DESCRIPTION
**Pull request by sazzadh88**
```addToCart``` function  doesn't check for stock available for selected variants. So it's required to check the entered ```quantity``` & stock available.

### CODE REVIEW

**Comment by sazzadh88** [comment]
sdfsdff

**Comment by sazzadh88** [comment]
@glennjacobs Please review it.

File: app/Http/Livewire/Components/AddToCart.php [hunk 1]
```diff
@@ -34,9 +34,12 @@ public function rules()
 
     public function addToCart()
     {
-        $this->validate();
-        CartSession::manager()->add($this->purchasable, $this->quantity);
-        $this->emit('add-to-cart');
+        if($this->purchasable->stock < $this->quantity){
```
Comment (alecritson) [inline app/Http/Livewire/Components/AddToCart.php:37]: I think here we can just return out of the first `if` statement, then we can remove the `else` and carry on as usual.

**Review by alecritson — CHANGES_REQUESTED** [review]
Thanks @sazzadh88 just a small point then I think this is good to go :)

File: app/Http/Livewire/Components/AddToCart.php (hunk 1 above)
Comment (sazzadh88) [reply app/Http/Livewire/Components/AddToCart.php:37]: Yes. Correct.

**Review by alecritson — APPROVED** [review]
Thanks @sazzadh88 will get this merged in :)

**Pull request merged**